tests/system/apmserver.py

```diff
@@ -164,9 +164,14 @@ class ServerSetUpBaseTest(BaseTest):
     sourcemap_url = "{}/{}".format(host, 'assets/v1/sourcemaps')
     expvar_url = "{}/{}".format(host, 'debug/vars')
 
+    jaeger_http_host = "localhost:14268"
```

Comment:
`localhost:14268` is the default host. To test that host configuration is properly picked up by the APM Server I would configure a different port. 
For testing with default values, you can remove this.

Reply:
The test setup now configures `apm-server.jaeger.http.host: "localhost:0"`, and parses the dynamically assigned port from the log file.